modules/test/playwright/tests/client-extension-web/env/portal-ext.properties

```diff
@@ -1 +1,3 @@
+feature.flag.LPD-27222=true
+
```

Comment:
No need for a line break between feature flag properties.

Reply:
If I remember well, SF did that. I will check it again